FirstProject/Program.cs

```diff
@@ -1,79 +1,108 @@
 ﻿using FirstProject;
 using System;
+using System.Drawing;
+using System.IO;
+
+string path = @"C:\Users\zodzy\Desktop\FirstProject\FirstProject\txt.txt";
+string text = "Witaj w moim kalkulatorze!";
```

Comment:
Czemu to wszystko jest w tym miejscu? przecież odczyt danych z pliku ma byćn ormalnie w menu po wyborze użytkownika